<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LOD Generator by Jonathan Flynn (i7241559)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="s_mainWindowGridLayout">
    <item row="0" column="1" alignment="Qt::AlignTop">
     <widget class="QGroupBox" name="s_transformGB">
      <property name="minimumSize">
       <size>
        <width>389</width>
        <height>507</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>389</width>
        <height>507</height>
       </size>
      </property>
      <property name="title">
       <string>LODGenerator</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QGroupBox" name="s_drawGB">
         <property name="title">
          <string>Edit LOD's</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="exportLODB">
            <property name="text">
             <string>Export LOD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteLODB">
            <property name="text">
             <string>Delete LOD</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="s_lodGB">
         <property name="title">
          <string>LOD's</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="s_loadF">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Loaded Object:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="fileNameL">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="s_pathF">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Path:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="pathLE">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="loadB">
               <property name="text">
                <string>Load</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QGroupBox" name="s_createLODGB">
         <property name="title">
          <string>Create LOD</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSpinBox" name="nFaces">
            <property name="minimum">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>#Faces</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createLODB">
            <property name="text">
             <string>Create LOD</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>nFaces</zorder>
         <zorder>createLODB</zorder>
         <zorder>label_3</zorder>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="exportAllB">
         <property name="text">
          <string>Export All</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QListWidget" name="m_lods">
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>BaseMesh</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Other Draw Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="m_wireframe">
            <property name="text">
             <string>Wireframe</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
